rust_src/src/eval_macros.rs

```diff
@@ -129,6 +129,84 @@ macro_rules! def_lisp_sym {
     ($name:expr, $value:expr) => {};
 }
 
+/// Macros we use to define forwarded Lisp variables.
+/// These are used in the syms_of_FILENAME functions.
+///
+/// An ordinary (not in buffer_defaults, per-buffer, or per-keyboard)
+/// lisp variable is actually a field in `struct emacs_globals'.  The
+/// field's name begins with "f_", which is a convention enforced by
+/// these macros.  Each such global has a corresponding #define in
+/// globals.h; the plain name should be used in the code.
+///
+/// E.g., the global "cons_cells_consed" is declared as "int
+/// f_cons_cells_consed" in globals.h, but there is a define:
```

Comment:
This isn't true of the Rust code, we use .f_Vfoo directly. We should update this paragraph or just remove it.